Tests/UnitTests/Core/Utilities/BinningDataTests.cpp

```diff
@@ -197,7 +197,7 @@ BOOST_AUTO_TEST_CASE(BinningData_bins) {
   BOOST_CHECK_EQUAL(yData_eq.searchLocal(xyPosition), size_t(1));
   BOOST_CHECK_EQUAL(zData_eq.searchLocal(rphizPosition), size_t(2));
   BOOST_CHECK_EQUAL(xData_arb.searchLocal(xyPosition), size_t(0));
-  BOOST_CHECK_EQUAL(xData_arb_binary.searchLocal(xyPosition), size_t(1));
+  BOOST_CHECK_EQUAL(xData_arb_binary.searchLocal(xyPosition), size_t(0));
```

Comment:
This was a buggy test covering a buggy result. `xData.bin(xyPosition)` has to result in bin 0, as it is `| 0 - 1|` and the the x value is 0.5,

Reply:
Ah, nice